openvm/tests/apc_builder.rs

```diff
@@ -117,5 +143,5 @@ is_valid * (is_valid - 1) = 0
 (id=6, mult=is_valid * 1, args=[b_msb_f_3 + 128, 0, 0, 0])
 "#;
 
-    assert_eq!(machine.to_string(), expected);
+    assert_eq!(expected, machine.to_string());
```

Comment:
This way, the diffs rendered by `assert_eq!()` make more sense (show you how to update the expectation).